_config.example.yml

```diff
@@ -0,0 +1,14 @@
+canvas-token: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+gitlab-host: https://git-classes.mst.edu/
+namespace: 2016-Fall-CS1001
+roster:
+- id: 000
```

Comment:
The `id` field is auto-pulled from GitLab; let's leave it out of the example since I'm pretty sure specifying the wrong one will do something weird.